js-exercises/minima/minima.js

```diff
@@ -0,0 +1,28 @@
+const sort = array => array.sort((a, b) => a - b);
```

Comment:
And what if complete list is not of numbers.
Program will not function properly if it get's String.
Tell developers, nothing other than number can come here.